src/frontend/src/utils/statusList/statuses/incompleteStatus.ts

```diff
@@ -24,27 +25,34 @@ export const incompleteStatus = (pia: IPiaForm | null) => {
         showSubmitButton: true,
         changeStatus: [
           {
-            status: 'EDIT_IN_PROGRESS',
+            status: 'EDIT_IN_PROGRESS', // change to EDIT_IN_PROGRESS
             modal: checkReviewStatus(pia)
               ? resetReviewEditInPRogressModal
               : defaultEditInProgressModal,
           },
           {
-            status: 'MPO_REVIEW',
-            modal: defaultMPOReviewModal,
+            status: 'MPO_REVIEW', // change to MPO_REVIEW
+            modal:
+              from === 'submit' ? submitPiaIntakeModal : defaultMPOReviewModal,
           },
           {
-            status: 'CPO_REVIEW',
+            status: 'CPO_REVIEW', // change to CPO_REVIEW
             modal: defaultCPOReviewModal,
           },
+          {
+            status: 'INCOMPLETE', // change to INCOMPLETE
```

Comment:
if you are in incomplete, you should not see an incomplete status in the drop down. can you double check this functionality is not broken with this change?